pandas/core/config_init.py

```diff
@@ -957,3 +960,53 @@ def register_converter_cb(key):
         styler_environment,
         validator=is_instance_factory([type(None), str]),
     )
+
+
+merge_chunk_size_doc = """
+: int | tuple[int, int]
+    Conditional merge chunked cross merge with deferred filter, chunk size.
+    If `int`, the int represents equal left and right chunksizes, and will be
+    converted to a tuple with 2 ints equal to the provided int. If `tuple`,
+    the tuple must be 2 ints, representing the left and right chunksizes.
+    All chunksizes must be greater than 0.
+"""
+
+
+def chunksize_validator(x: Any):
+
+    exc = ValueError(
+        "option must be an int greater than 0, or a tuple of 2 ints, "
+        "both greater than 0"
+    )
+    if isinstance(x, int):
+        if x <= 0:
+            raise exc
+    elif isinstance(x, tuple):
+        if len(x) != 2:
+            raise exc
+        for i in x:
```

Comment:
Nit: `if any(not (isinstance(i, int) and i > 0) for i in x):`

Reply:
done